### ISSUE TITLE
Basic Questions on Usage

### ISSUE THREAD

**Issue by gwmatthews** [polymny/polymny#21]
I have some basic questions about this project that were unclear in the English language documentation. 

Does this system need to be setup on a server pushing videos made from pdf slideshows and a webcam? In other words do I need dedicated server space to use this?

Or is this a video production method that enables me to create video files as outputs which I can then upload to an existing site, such as Wordpress or Moodle?

**Comment by tforgione** [comment]
Yes, the majority of the people interested in this project is french, so we didn't spend much time _yet_ on the english translation, sorry for that. It is something we think about though, we have #16 opened for that purpose.

You don't need a dedicated server to use polymny, you should be able to simply go to [polymny.studio](https://polymny.studio), upload your slides and start recording. Then, you'll be able to set the parameters for the production of the video and publish it. All the encoding will be on our server. When you publish the video on polymny, it is encoded and hosted on our server, so you just get a link like this that you can share : https://video.polymny.studio/?v=b4a86be5-eb21-4681-8716-b96458e60cfe/

You can still download the video when produced if you want to upload it somewhere, but there is also the possibility to embed an iframe with our video player.

**Comment by gwmatthews** [comment]
Wonderful, thanks for the clarification. I managed to sign up now and can see more clearly what is involved! I'll have to work on my French by watching some tutorials.
